services/datamanager/builtin/builtin.go

```diff
@@ -426,12 +428,16 @@ func (svc *builtIn) Reconfigure(
 	}
 	svc.collectors = newCollectors
 
+	if svc.syncDisabled == svcConfig.ScheduledSyncDisabled &&
+		svc.syncIntervalMins == svcConfig.SyncIntervalMins &&
+		reflect.DeepEqual(svc.additionalSyncPaths, svcConfig.AdditionalSyncPaths) {
```

Comment:
I don't think the additional paths changing needs to impact the sync ticker. More of a product question ig but I'd lean towards removing this part of the check